docs/en/stack/ml/df-analytics/ml-trained-models.asciidoc

```diff
@@ -28,6 +20,20 @@ Alternatively, you can use APIs like
 {ref}/get-trained-models.html[get trained models] and
 {ref}/delete-trained-models.html[delete trained models].
 
+[discrete]
+[[deploy-dfa-trained-models]]
+== Deploying trained models
+
+include::ml-dfa-shared.asciidoc[tag=dfa-deploy-model]
+
+You can also supply trained models that are not created by {dfanalytics-job} but
```

Comment:
The changes overall LGTM but I want to be clear that this functionality right now is specific to DFA regression and classification trained models.

Reply:
Thank you @alvarezmelissa87! I further emphasized that the added content applies only to DFA models via [64aedcb](https://github.com/elastic/stack-docs/pull/2524/commits/64aedcb902b7c9c09da48ae2671fdf4334893dab).